tests/drivers/flash_simulator_stub/boards/qemu_x86_ev_0x00.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&sim_flash {
+	erase-value = < 0x00 >;
```

Comment:
why 0x00 ?

Reply:
Because this test variant is for 0x00 erased flash.